### PR TITLE
Lunch menu UI black box test

### PR DESCRIPTION
**Pull request by dbacinski**

### CODE REVIEW

**Comment by Tomekkk** [comment]
+1

File: app/src/androidTest/groovy/com/macoscope/ketchuplunch/RecyclerViewMatcher.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ private boolean cacheItemView(View view) {
             if (itemView == null) {
                 RecyclerView recyclerView = findRecyclerViewById(view);
 
-                if (recyclerView == null) {
+                if (recyclerView == null || recyclerView.getAdapter().getItemCount() == 0) {
```
Comment (pziomacki) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/RecyclerViewMatcher.java]: Is it possible here for recyclerView.getAdapter() to be null?

**Comment by pziomacki** [comment]
+2

File: app/src/androidTest/groovy/com/macoscope/ketchuplunch/RecyclerViewMatcher.java (hunk 1 above)
Comment (dbacinski) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/RecyclerViewMatcher.java]: right, possible NPE here, have to fix it

Comment (dbacinski) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/RecyclerViewMatcher.java]: yes

**Pull request merged**